src/components/table/schedule/schedule-parts/base-row.component.tsx

```diff
@@ -102,9 +101,6 @@ export function ScheduleRowComponent({
 
     if (event.ctrlKey && DirectionKey[event.key]) {
       !selectionMode && setSelectionMode(true);
-      // TODO: refactor. It solves problem with wrong elements selection on the ends when direction changes
```

Comment:
Raczej taki refactor był by zbędny, bo i tak to całkiem dobrze działa i też nie ma znaczenia w większej skali. I jedyny problem który z tym był to to że nieintuicyjnie wyglądał